README.md

```diff
@@ -94,14 +94,15 @@ This automatically runs the `fr.openfisca.org/api/deploy-latest.sh` script.
   - run `sudo su - openfisca`
   - pull the new `openfisca-ops`
   - get to the virtualenvs folder (`cd /home/openfisca/virtualenvs`)
-  - create the new virtualenv (`virtualenv api-frxx`)
+  - create the new virtualenv (`virtualenv --python /home/openfisca/.pyenv/versions/3.7.0/bin/python api-frxx`)
```

Comment:
> Explicit why `pyenv` is used here? Is it because on Debian, Python 3.7 isn't distributed?

Reply:
I agree we could add somewhere that we use `pyenv`. Not blocking for this PR, however.